_translations/en.yml

```diff
@@ -586,9 +586,9 @@ en:
     title: "Bitcoin - Open source P2P money"
     metadescription: "Bitcoin is an innovative payment network and a new kind of money. Find all you need to know and get started with Bitcoin on bitcoin.org."
     listintro: "Bitcoin is an innovative payment network and a new kind of money."
-    list1: "Fast peer-to-peer<br>transactions"
+    list1: "Uncensorable peer-to-peer<br>transactions"
```

Comment:
How about: `Decentralized peer-to-peer transactions` or just `Peer-to-Peer (P2P) transactions`?

Reply:
Sure, if anything I'd just simplify it to peer-to-peer but my main concern is that this may be meaningless to most folks.

Reply:
Maybe "Transactions with no middlemen required"?

Reply:
What about word "direct"? "Direct payments" should be easy to understand by most people.

Reply:
I like that, or perhaps "Direct person-to-person payments"

Reply:
@jlopp Let's update it to one of the recent suggestions above and then we can merge this.